docs/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md

```diff
@@ -31,16 +31,17 @@ To make a new branch and submit it for review, create a GitHub pull request with
 1. Check out the `master` branch with `git checkout master`.
 2. Retrieve new changes to the `master` branch with `brew update`.
 3. Create a new branch from the latest `master` branch with `git checkout -b <YOUR_BRANCH_NAME> origin/master`.
-4. Make your changes to any Homebrew formula with `brew edit` or your favorite text editor, following all the guidelines in the [Formula Cookbook](Formula-Cookbook.md).
```

Comment:
Maybe just add an `e.g.` in here rather than a new line. Want to try and keep this as compact as possible.